docker/README.md

```diff
@@ -22,7 +22,7 @@ Optional
 - `REDIS_PASSWORD` - a Redis password (if you need one)
 - `REDIS_DB_NUMBER` - a Redis database number (default: `0`)
 - `REDIS_NAMESPACE` - a Redis [namespace][namespace] name (if you have separated sidekiq)
-- `REDIS_NO_CLIENT_NAME` - informs Sidekiq not to set a client name for the Redis connection (this might be required if a SaaS Redis provider is being used, which blocks the `CLIENT` command)
+- `REDIS_DISABLE_CLIENT_ID` - informs Sidekiq not to set a client name for the Redis connection (this might be required if a SaaS Redis provider is being used, which blocks the `CLIENT` command)
```

Comment:
Can you also fix a description. Since it's not a client name but id

Reply:
Should be fixed.